pkg/controller/migstorage/migstorage_controller.go

```diff
@@ -107,26 +108,33 @@ func (r *ReconcileMigStorage) Reconcile(request reconcile.Request) (reconcile.Re
 	err = r.Get(context.TODO(), request.NamespacedName, storage)
 	if err != nil {
 		if errors.IsNotFound(err) {
+			log.V(2).Info("unable to get migstorage, not found", "migstorage", request.String(), "error", err.Error())
```

Comment:
1. Was this `V(2)` / `V(4)` functionality already built into the logger and we just weren't using it?
2. How does the user control the level of log messages they will see?

Reply:
Can we please use good grammar in log messages?  Begin with capital and end with period.

Reply:
> Since the MTC operator controls the container environment and by default does not restrict logging level, is this change necessary? How would the logging level get changed resulting in suppression?

I believe turning the flag here to true https://github.com/konveyor/mig-controller/blob/ce7c0b8bc1255f991c94cc4566da176d23aefc35/cmd/manager/main.go#L41 will turn debug level logging on. My understanding on the loggin numbers was different than what I found in the vendored code. it starts with debug level at -1 https://github.com/konveyor/mig-controller/blob/3e8f9fe72c3a5a3fec64b3a0b1bbb0e7101fc544/vendor/go.uber.org/zap/zapcore/level.go#L37

> Any reason to not use good grammar in log messages? Begin with capital and end with period.

I thought that it was a convention in Go to not begin with a Capital. But TIL that's only for errors. I'll refactor this wherever applies.